src/components/Dropdown/DropdownPills.tsx

```diff
@@ -76,7 +80,11 @@ export const DropdownPills = ({
     return null;
 
   return (
-    <ul className='o-multiselect_choices pills'>
+    <ul
+      className={`o-multiselect_choices pills${
+        pillAlign === 'bottom' ? ' bottom' : ''
```

Comment:
And then add the modifier class here:

```suggestion
      className={`o-multiselect_choices${
        pillAlign === 'bottom' ? ' o-multiselect_choices__bottom' : ''
```